src/test/java/test/CustomPortTest.java

```diff
@@ -0,0 +1,47 @@
+package test;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.TestInfo;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.EnumSource;
+import test.helm.Helm;
+import test.model.Container;
+import test.model.Product;
+
+import java.util.Map;
+
+import static test.jackson.JsonNodeAssert.assertThat;
+
+class CustomPortTest {
+    private Helm helm;
+
+    @BeforeEach
+    void initHelm(TestInfo testInfo) {
+        helm = new Helm(testInfo);
+    }
+
```

Comment:
Is not crucial but I think maybe good to have unit test for default values for `ATL_TOMCAT_PORT` in different products

Reply:
The default values are covered usually by the `HelmOutputComparisonTest` and using the expected `output.yaml` files - this is an example:
https://github.com/atlassian/data-center-helm-charts/pull/419/files#diff-63ea51911cc0f4498740fb7f8ee9bba6524a8c59bf6378ae51489847c9178444R268